plugin/src/main/groovy/gorm/tools/repository/EntityFieldsHandler.groovy

```diff
@@ -0,0 +1,32 @@
+package gorm.tools.repository
+
+import gorm.tools.beans.BeanPathTools
+import grails.core.GrailsApplication
+import org.springframework.beans.factory.annotation.Autowired
+
+trait EntityFieldsHandler<D> {
+
+    @Autowired
+    GrailsApplication grailsApplication
+
+    abstract Class<D> getEntityClass()
+    List<String> getSelectFieldsConfig(){getFieldsConfig('select')}
```

Comment:
having a getFieldsConfig should be adequate. lets remove these specific ones here